engine/settings/base.py

```diff
@@ -667,6 +667,7 @@ class BrokerTypes:
 DRF_RECAPTCHA_DEFAULT_V3_SCORE = 0.5
 DRF_RECAPTCHA_TESTING = True
 DRF_RECAPTCHA_TESTING_PASS = True
+RECAPTCHA_SITE_KEY = os.environ.get("RECAPTCHA_SITE_KEY", default="6LeIPJ8kAAAAAJdUfjO3uUtQtVxsYf93y46mTec1")
```

Comment:
I would still recommend removing `FEATURE_CAPTCHA_ENABLED` and `RECAPTCHA_SECRET_KEY` up above to avoid confusion. what do you think?